homeassistant/components/generic/camera.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, hass, device_info):
         if self._stream_source is not None:
             self._stream_source.hass = hass
         self._limit_refetch = device_info[CONF_LIMIT_REFETCH_TO_URL_CHANGE]
-        self._attr_frames_interval = 1 / device_info[CONF_FRAMERATE]
+        self._attr_frame_interval = 1 / device_info[CONF_FRAMERATE]
```

Comment:
Should there be a test for this feature?

Reply:
Yes, there [was a test previously ](https://github.com/home-assistant/core/pull/63197#discussion_r777190855) but it was removed. I will re-add it.